### PR TITLE
Updating the release notes

### PR DESCRIPTION
**Pull request by kaerm**
storage was split by event hubs

### CODE REVIEW

File: releases/2020-05/python.md [hunk 1]
```diff
@@ -53,12 +53,16 @@ If you have a bug or feature request for one of the libraries, please post an is
...
 ## Changelog
 
-### Azure Storage
+### Storage
 
 #### Blob [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/CHANGELOG.md)
 
 - This release contains bug fixes to improve the quality of the library
 
+#### File Data Lake [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-file-datalake/CHANGELOG.md)
```
Comment (Petermarcu) [inline releases/2020-05/python.md:62]: Do we call it File Data Lake or just Data Lake?

Comment (Petermarcu) [reply releases/2020-05/python.md:62]: https://azure.microsoft.com/en-us/services/storage/data-lake-storage/

Comment (kaerm) [reply releases/2020-05/python.md:62]: Not sure, I just moved the section that already existed. Checked the docs for other languages and everywhere we refer to it as File Data Lake. @rishabpoh any guidance?

Comment (xiangyan99) [reply releases/2020-05/python.md:62]: I see we use "Storage File Datalake" in https://github.com/Azure/azure-sdk/blob/master/releases/2020-03/python.md & https://github.com/Azure/azure-sdk/blob/master/releases/2020-02/python.md

Comment (kaerm) [reply releases/2020-05/python.md:62]: Okay, I will go ahead and merge this for now to fix the existing issues if I don't hear any objections. If we decide that we need to update the way we call Data Lake we will have to scrub other places anyway.

**Pull request merged**